pmacApp/powerPmacAsynPortSrc/sshDriver.cpp

```diff
@@ -16,6 +16,14 @@
 #include <osiUnistd.h>
 #include <osiSock.h>
 #include <poll.h>
+#include <stdlib.h>
+
+#define SSH_PORT 22
+#define SSH_NOT_CONNECTED 0
+#define SSH_CONNECTED 1
+#define SSH_ERROR_TIMEOUT_MS 100
+
+#define PPMAC_MAX_CHAR_LENGTH 8192
```

Comment:
In PowerPMAC Software Reference Manual, we have the following release note for V2.6.1 firmware.

> Now reject command lines with over 7168 characters with ERR28. Previously could
crash Power PMAC.

Would it make sense to define `PPMAC_MAX_CHAR_LENGTH` to 7168 instead?

Reply:
Thanks @LeandroMartinsdS I hadn't found any text relating to that limit.  I tested up to 8192 but potentially not on that firmware and so yes we should reduce the LENGTH to 7168, I will do that.